styles/base.css

```diff
@@ -0,0 +1,98 @@
+/*some css for mobile view*/
+*{
+  font-family: sans-serif;
+}
+
+a{
+  color: inherit;
+  text-decoration: none;
+}
+div{
+  height: 100px;
+  margin-top: 50px;
+  border: 1px solid black;
+}
+.my-name{
+  margin: auto;
+  height: inherit;
+  display: inline-block;
+  border: none;
+  transform: translateY(25%);
+  font-size: 20px;
+  font-weight: bolder;
+  margin-left: 2%;
+}
+.name{
+  height: 100px;
+}
+.hidden{
+  display: none;
+}
+#main-nav ul{
+  display: block;
+  float: right;
+  list-style-type: none;
+  text-align: right;
+  font-weight: bold;
+  font-size: 0.8em;
+  background: #f1f1f1;
+  font-size: 19px;
+  margin: auto;
+  padding: 4%;
+  /* margin-right: -10%; */
+}
+
+.header{
+  height: 50px;
+  background-color: grey;
+}
+/*#main-nav:hover ul {
```

Comment:
IF you do not nee these, I would suggest removing them. We do not want commented out code in master!